jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```diff
@@ -138,24 +127,28 @@ YUI.add('env-switcher', function() {
       // XXX j.c.sackett 2015-10-28 When we have the UI for template creation
       // and template selection in the GUI, this code should be updated to not
       // select template based on state server.
-      var srvCb = function(servers) {
-        if (servers && servers.length > 0 && servers[0].path) {
-          var serverName = servers[0].path;
-          var baseTemplate = serverName;
-          jem.newEnvironment(
-            envOwnerName, envName, baseTemplate, serverName, password,
-            this.createEnvCallback, this.jemFailHandler);
+      var srvCb = function(error, servers) {
+        if (error) {
+          console.log(error);
```

Comment:
I'd prefer the 'no else' syntax here, so you would explicitly return from the `if (error) {` and then fall through for the rest of the code.